docs/get_started.rst

```diff
@@ -1,17 +1,353 @@
+|                                                                  | Enter the name of the package you want to use.                                                                                                                                                                                                                                                        |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Is this a managed package project?                               |                                                                                                                                                                                                                                                                                                       |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Salesforce API Version                                           | Which Salesforce API version does your project use? Defaults to the latest.                                                                                                                                                                                                                           |
++------------------------------------------------------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Which [source format] do you want to use?                        | Metadata API format is the "older" format and stores data under the `src/` directory.                                                                                                                                                                                                                 |
+|                                                                  | DX source format (aka "SFDX Format") stores data under the `force-app/` directory.                                                                                                                                                                                                                    |
```

Comment:
The actual options they can enter here are `sfdx` and `mdapi`. We should use those here in the docs.